locale/es.po

```diff
@@ -622,7 +622,7 @@ msgstr "Ninguno"
 #, python-brace-format
 msgid "{name} {version} {package_source} package is up to date - skipping"
 msgstr ""
-"El paquete {name} {version} {package_source} está actualizado - abandono"
+"El paquete {name} {version} {package_source} está actualizado - ignorado"
```

Comment:
i think it would make it too similar with the message about package being ignored (via --ignore or IgnorePkgs in pacman.conf) - is there a better word for `skipping`?

Reply:
Yes, nice catch. I think there is not a direct translation with one work to "skipping". If skipping exactly means "skipping to the next package", it could be translated literally as "pasando al siguiente", but maybe it's too long. If not, that would be the exact translation. At least in Spanish from Spain. I'm sure south american spanish speaking would translate differently, but they'll surely understand. I can change it as you find it more accurate.

Reply:
yup, let's try pasando, because just ignorado seems like it could mislead due to wording with another similar state

Reply:
(mb eventually i wouldn't need to just make more clear messages in english at first place, to create less such potential translation collisions but i'm really bad in forming the things/ideas into human-readable words 😸)

Reply:
OK, I'll change the message as "pasando al siguiente" (just "pasando" sounds weird in Spanish, it is like dismissing something in a bad way).